src/test/robot/api/remote/run.robot

```diff
@@ -38,6 +42,13 @@ Should Stop Running Scenario
     ${resp_stop} =  Stop Mongoose Scenario Run  ${resp_etag_header}
     Should Be Equal As Strings  ${resp_stop.status_code}  200
 
+Should Stop Running Scenario In Distributed Mode
+    ${data} =  Make Start Request Payload For Distributed Mode
+    ${resp_start} =  Start Mongoose Scenario  ${data}
+    ${resp_etag_header} =  Get From Dictionary  ${resp_start.headers}  ${HEADER_ETAG}
+    ${resp_stop} =  Stop Mongoose Scenario Run  ${resp_etag_header}
```

Comment:
Is there a way for us to check that the container is finished? As you remember, the bug actually was returning 200 status code, but not actually stopping mongoose

Reply:
I will check Mongoose logs for this Test